harness/determined/cli/user.py

```diff
@@ -86,7 +58,6 @@ def log_out_user(parsed_args: Namespace) -> None:
     auth = authentication.cli_auth
     if auth is None:
         return
-
     try:
         api.post(
```

Comment:
can we please update this logout call as well? and any other `api.post`, `api.get`, `api.patch` remaining in this file, if any.